.github/workflows/e2e.yml

```diff
@@ -53,6 +53,10 @@ on:
         type: boolean
         required: false
         default: false
+      v2-migration-test:
+        type: boolean
+        required: false
+        default: false
```

Comment:
This brings us to 11 `inputs` now. The limit is 10: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#providing-inputs

We could move to taking a single comma separated input an then splitting it in the matrix-conditionals step.

Reply:
https://github.com/zeta-chain/node/pull/2777 will resolve this regardless of what we are running